spark/v4.0/spark/src/test/java/org/apache/iceberg/spark/SparkCatalogConfig.java

```diff
@@ -48,6 +48,11 @@ public enum SparkCatalogConfig {
           "cache-enabled",
               "false" // Spark will delete tables using v1, leaving the cache out of sync
           )),
+  /**
+   * @deprecated wll be removed in 2.0.0, use {@link SparkCatalogConfig#SPARK_SESSION} instead.
+   */
+  @Deprecated
```

Comment:
this is test code, so there's no need to deprecate anything

Reply:
Is it possible for downstream projects to use `SparkCatalogConfig`?